libraries/AP_NavEKF3/AP_NavEKF3_PosVelFusion.cpp

```diff
@@ -589,12 +590,9 @@ void NavEKF3_core::SelectVelPosFusion()
 void NavEKF3_core::FuseVelPosNED()
 {
     // health is set bad until test passed
-    bool velHealth = false; // boolean true if velocity measurements have passed innovation consistency check
-    bool posHealth = false; // boolean true if position measurements have passed innovation consistency check
-    bool hgtHealth = false; // boolean true if height measurements have passed innovation consistency check
-
-    // declare variables used to check measurement errors
-    Vector3f velInnov;
+    bool velCheckPassed = false; // boolean true if velocity measurements have passed innovation consistency checks
+    bool posCheckPassed = false; // boolean true if position measurements have passed innovation consistency check
+    bool hgtCheckPassed = false; // boolean true if height measurements have passed innovation consistency check
 
     // declare variables used to control access to arrays
     bool fuseData[6] = {false,false,false,false,false,false};
```

Comment:
can just be:
bool fuseData[6] {};
(as false is zero)

Reply:
Done